src/posts/articles/impact-of-tax-changes-2025-2026.md

```diff
@@ -4,12 +4,12 @@ This analysis, requested and [published](https://insider.iea.org.uk/p/new-analys
 
 ## Policy changes
 
-From a combination of temporary reforms expiring and new tax reforms taking effect
+From a combination of temporary reforms expiring and new tax reforms taking effect:
 
-- The Government will [set](https://www.legislation.gov.uk/ukpga/2023/2/section/1) **Stamp Duty Land Tax** nil-rate thresholds at £300,000 (from £425,000) for first-time buyers and £125,000 (from £250,000) for subsequent purchases. These thresholds determine the property value at which SDLT begins to apply. First-time buyers maintain a higher threshold than subsequent purchasers under this reform.
-- The Government will [set](https://bills.parliament.uk/bills/3888) **Employer's National Insurance** rate will [change](https://policyengine.org/uk/research/autumn-budget-24-employer-ni) to 15% with the secondary threshold set to £96.14 (from £175) per week. We assume that the employer NI changes will be passed on to employees at 40% in 2025, 50% in 2026, 60% in 2027, and 70% in 2028-2029, with the remainder split between business owners and prices.
+- The Government will [set](https://www.legislation.gov.uk/ukpga/2023/2/section/1) **Stamp Duty Land Tax** nil-rate thresholds at £300,000 (from £425,000) for first-time buyers and £125,000 (from £250,000) for subsequent purchases. These thresholds determine the property value at which SDLT begins to apply. First-time buyers maintain a higher threshold than subsequent purchasers under this reform. This reform is the result of an expiry of a Stamp Duty Land Tax cut announced by then-Chancellor Jeremy Hunt in the Autumn Budget 2023 (expiring 31 March 2025).
```

Comment:
ok...i wouldn't really say the government is setting the rate if they're just allowing a previously scheduled expiration to occur (per our current law/current policy discussion this morning). can we describe these as tax changes instead?

Reply:
Yes sure though no active voice then

Reply:
Actually amended- still active